### PR TITLE
Replace 20.04/Focal Fossa by 22.04/Jammy Jellyfish

### PR DESCRIPTION
**Pull request by simondeziel**

### CODE REVIEW

File: doc/clustering.md [hunk 1]
```diff
@@ -634,7 +634,7 @@ This is done by using the `@` prefix when using `--target`.
 
 An example:
 ```bash
-lxc launch images:ubuntu/20.04 cluster:ubuntu --target=@gpu
+lxc launch images:ubuntu/22.04 cluster:ubuntu --target=@gpu
```
Comment (tomponline) [inline doc/clustering.md:637]: Should this be `ubuntu:22.04`?

File: doc/howto/network_ovn_setup.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ Create an OVN network and an instance using it:
 ```
 lxc network set lxdbr0 ipv4.dhcp.ranges=... ipv4.ovn.ranges=... # Allocate IP range for OVN gateways.
 lxc network create ovntest --type=ovn network=lxdbr0
-lxc init images:ubuntu/20.04 c1
+lxc init images:ubuntu/22.04 c1
```
Comment (tomponline) [inline doc/howto/network_ovn_setup.md:24]: Should this be ubuntu:22.04?

File: doc/clustering.md (hunk 1 above)
Comment (simondeziel) [reply doc/clustering.md:637]: Both work but in doc/clustering.md, the official Ubuntu image is used already so this is not consistent. Not sure which we should align to.

Comment (tomponline) [reply doc/clustering.md:637]: It looks like we are using `ubuntu:...` most other places, except for when we are using the `/cloud` variants.

File: doc/howto/network_ovn_setup.md (hunk 2 above)
Comment (simondeziel) [reply doc/howto/network_ovn_setup.md:24]: I would think the community image would work here. If you prefer the official one, it's OK with me.

Comment (tomponline) [reply doc/howto/network_ovn_setup.md:24]: It looks like we are standardizing on the `ubuntu:...` image most other places.

File: doc/clustering.md (hunk 1 above)
Comment (tomponline) [reply doc/clustering.md:637]: Same

**Comment by stgraber** [comment]
Needs to be sliced up differently.
Can you make:
 - One commit for the lxc/ stuff
 - One commit for the doc/ stuff minus rest-api.yaml
 - One commit for the shared/api stuff
 - One commit for the lxd/db stuff
 - One commit for the swagger update (`make update-api`)
 - One commit for the i18n update (`make i18n`)

**Comment by simondeziel** [comment]
> Needs to be sliced up differently. Can you make:

Done, I didn't know about `make update-api`!

**Pull request merged**